### PR TITLE
Remove init command

### PR DESCRIPTION
**Pull request by wtrocki**
## Motivation

We no longer have templates repository. 
Using graphback as tool to the bootstrap project is deprecated.
With that, we need to remove init command, but still allow config command to be present. 

Config command will help to apply config changes to existing projects.
GraphQL CLI will be used for init purposes along with it's templates.

Working on docs to reflect that change

### CODE REVIEW

**Comment by wtrocki** [comment]
I have done some very basic changes to the docs in order to publish them after 0.10.0 release

File: docs/gettingstarted.md [hunk 1]
```diff
@@ -4,11 +4,46 @@ title: What is Graphback and how do I get started?
 sidebar_label: What is Graphback
 ---
 
-Graphback is a CLI tool that can help you bootstrap a GraphQL server and client for you including GraphQL schema, resolvers and generating database structure. Graphback addresses the difficulty in configuring a GraphQL server from scratch.
+Graphback is an package andCLI tool that can help you bootstrap a GraphQL server and client for you including GraphQL schema, resolvers and generating database structure. Graphback addresses the difficulty in configuring a GraphQL server from scratch by giving developers:
```
Comment (craicoverflow) [inline docs/gettingstarted.md]:
```suggestion
Graphback is a package and CLI tool that can help you bootstrap a GraphQL server and client for you including GraphQL schema, resolvers and generating database structure. Graphback addresses the difficulty in configuring a GraphQL server from scratch by giving developers:
```

File: docs/gettingstarted.md [hunk 2]
```diff
@@ -4,11 +4,46 @@ title: What is Graphback and how do I get started?
...
 
-<div style="width:100%;height:0;padding-bottom:50%;position:relative;"><iframe src="https://giphy.com/embed/Uov4EC4W3G74TtWZKf" width="100%" height="100%" style="position:absolute" frameBorder="0" class="giphy-embed" allowFullScreen></iframe></div>
+- Generated GraphQL Schema containing Queries and Mutations for general data access
+- Underlying resolvers to fetch data in performant way
+- Support for subscriptions and live updates on the data
+- Relational support
+- Database migrations 
+- Extendable runtime layer for composing data services used in generated resolvers
```
Comment (craicoverflow) [inline docs/gettingstarted.md]:
```suggestion
- Extensible runtime layer for composing data services used in generated resolvers
```

File: docs/gettingstarted.md [hunk 3]
```diff
@@ -4,11 +4,46 @@ title: What is Graphback and how do I get started?
...
+
+Execute init command to create project from scratch
+
+```
+graphql init
+```
+
+Please answer all questions.
+After project is created please follow project README file that might differ depending on
+the template tha was choosen on startup.
```
Comment (craicoverflow) [inline docs/gettingstarted.md]:
```suggestion
the template that was chosen on startup.
```

File: docs/gettingstarted.md [hunk 4]
```diff
@@ -4,11 +4,46 @@ title: What is Graphback and how do I get started?
...
+
+GraphQL CLI offers various commands to interact with your project that work along Graphback package.
```
Comment (craicoverflow) [inline docs/gettingstarted.md]:
```suggestion
GraphQL CLI offers various commands to interact with your project that work with the Graphback package.
```

File: docs/gettingstarted.md [hunk 5]
```diff
@@ -4,11 +4,46 @@ title: What is Graphback and how do I get started?
...
+Graphback specific commands are available by executing `graphql generate` 
+ 
+## Adding Graphback to existing project using Graphback CLI
 
-## Quick Start: Installing and running Graphback
 ```bash
 npm i -g graphback-cli
 graphback init <project-name>
```
Comment (craicoverflow) [inline docs/gettingstarted.md]: Aren't we removing the `init` command?

File: docs/releases.md [hunk 6]
```diff
@@ -5,9 +5,46 @@ title: Releases
 
 ## What's new in Graphback
 
+
 This file contains changes and migration steps for Graphback project. 
 Please follow indidividual releases for more information.
 
+
+# 0.10.0
```
Comment (wtrocki) [inline docs/releases.md]: @craicoverflow I would love to get your updates for that as well. What we have changed since 0.9.0 :D

**Pull request merged**